tools/generate_codemeta.py

```diff
@@ -0,0 +1,112 @@
+import os
+import subprocess
+
+# UPDATE THESE AS NEEDED WITH EACH RELEASE
+release_date = '2020-12-17'
+release_version = '0.22.0'  # major.minor.patch; must include trailing zero
```

Comment:
Can't these use `datetime.today().strptime` and `import mne; mne.__version__` and then assert that the version is  formatted as `X.Y.Z` (no `dev`) for example?